.github/ISSUE_TEMPLATE/new-hub.yml

```diff
@@ -0,0 +1,164 @@
+        For example, if it will be used for a class, the starting day of class and any days with tests.
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Hub Customization
+
+  - type: input
+    id: auth-type
+    attributes:
+      label: Hub Authentication Type
+      description: |
+        What kind of authentication service will be used for this community.
+        This boils down to what kind of "log-in username" users wish to use on the hub.
+        Currently, the two options are **GitHub handles** (e.g., `@mygithubhandle`) or Google OAuth usernames (e.g., `myname@2i2c.org`).
+        You must choose **only one option**.
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: features
+    attributes:
+      label: Extra features you'd like to enable
+      description: |
+        In addition to a base JupyterHub, you may enable extra infrastructure features that provide additional functionality.
+        These generally come with added complexity and cost, so make sure you've discussed these options with somebody at 2i2c before selecting them here.
+
+      options:
+        - label: Dedicated Scalable Dask Cluster with [Dask Gateway](https://gateway.dask.org/)
+    validations:
+      required: false
+
+  - type: input
+    id: hub-logo
+    attributes:
+      label: Hub logo
+      description: |
+        A URL to an image that will be used on the hub's landing page. (optional)
+    validations:
+      required: true
+
+  - type: input
+    id: hub-logo-url
+    attributes:
+      label: Hub logo URL
+      description: |
+        A URL that the hub's logo will point to when users click on it on the landing page (optional)
+        
+    validations:
+      required: false
+
+  - type: input
+    id: hub-image-service
+    attributes:
+      label: Hub image service
+      description: |
+        The image registry service used for this hub's user image, if a custom user image is desired.
+        For example, a [quay.io registry image](https://quay.io/).
+        See [](customize/custom-image) for more information.
+        
+    validations:
+      required: false
+
+  - type: input
+    id: hub-image
+    attributes:
+      label: Hub image
+      description: |
+        A name and tag for a user image, pointing to the registry in the field above.
+        A name and tag for a user image, pointing to the registry in the field above.  
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Hub Engineer information
+
+        The Hub Engineer should fill in the metadata below when it available.
+        The Community Representative shouldn't worry about this section, but may be asked to provide help answering some questions.
+        
+        ### Deployment information
+
+        **Hub ID:**
+          The unique ID used to identify this hub. Should be URL-friendly and short. (e.g. `my-university`).
```

Comment:
```suggestion
          <!-- The unique ID used to identify this hub. Should be URL-friendly and short. (e.g. `my-university`). -->
```

I would make the descriptions for the fields in the Engineer section HTML comments, so it looks a bit neater when rendered.